### ISSUE TITLE
Isn't tailwind included by default? 

### ISSUE THREAD

**Issue by notflip** [cossssmin/jigsaw-tailwindcss#39]
Or am I wrong? The default html that I'm seeing in my blog template does look a lot like tailwind.

**Comment by bingalls** [comment]
Yes, https://github.com/tighten/jigsaw/blob/main/stubs/site/package.json currently shows Tailwind v2, newer than tailwind in this project.
